specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -215,6 +215,13 @@ export const userPreferenceDefinitions = {
             renderer: ColorPickerPreferenceItem,
             container: 'label',
           }),
+          lightSideBarBackground: defineItem({
+            title: preferencesText.lightSideBarBackground(),
+            requiresReload: false,
+            visible: isLightMode,
+            defaultValue: false,
+            type: 'java.lang.Boolean',
+          }),
```

Comment:
instead of making this a boolean, could we have it be one of the two values:
- "Dark"
- "Match theme color"

Benefits:
- If it is not a boolean, but a multiple choice, it's much easier to extend it in the future with additional types
- You don't have to handle the case when theme color is dark but side bar color is light (unless we want that to be a feature)